test/engineEvaluate.test.ts

```diff
@@ -0,0 +1,132 @@
+import { testables } from '../src/engine/evaluate';
+import {
+  makeEngineCondition,
+  makeQuery,
+  makePageView,
+} from './helpers/engineConditions';
+import { QueryFilterComparisonType } from '../types';
+
+const { filterPageViews } = testables;
+
+const multipleOrthogonalQueriesCondition = makeEngineCondition(
+  [
+    makeQuery(
+      {
+        vector: [1, 0, 0],
+        threshold: 0.99,
+      },
+      1,
+      QueryFilterComparisonType.COSINE_SIMILARITY
+    ),
+    makeQuery(
+      {
+        vector: [0, 1, 0],
+        threshold: 0.99,
+      },
+      1,
+      QueryFilterComparisonType.COSINE_SIMILARITY
+    ),
+  ],
+  1
+);
+
+const multipleLinearlyDependentQueriesCondition = makeEngineCondition(
```

Comment:
Hmm not sure this is really explaining what this is in this context. 

Also where is the 3rd vector - where the combination can be used to make another vector.

Reply:
Missing! My bad.
`multipleOverlapingQueryConditions` would be a better naming, WDYT?